vectors.go

```diff
@@ -0,0 +1,284 @@
+package pinecone
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+)
+
+// DescribeIndexStatsParams represents the parameters for a describe index stats request.
+type DescribeIndexStatsParams struct {
+	Filter map[string]any `json:"filter"`
```

Comment:
```suggestion
	Filter map[string]string `json:"filter"`
```

![image](https://user-images.githubusercontent.com/11081491/236595422-2b649d92-5622-4af5-a030-7258e7a70f5d.png)

Reply:
<img width="835" alt="Screenshot 2023-05-06 at 16 54 08" src="https://user-images.githubusercontent.com/32185409/236646202-5bec9720-930b-4026-8a6f-cdbaf0494cd1.png">


This part in the [HERE](https://docs.pinecone.io/docs/metadata-filtering) suggests that you can use more than strings as values

Reply:
Oh, looks like I missed this documentation. Thank you for referencing it!